src/main/java/org/jalasoft/moi/controller/config/SwaggerConfig.java

```diff
@@ -30,34 +35,35 @@
 public class SwaggerConfig {
 
     /**
-     * Docket is the primary api configuration mechanism is initialized for swagger specification 2.0.
+     * Documents information the app RESTful web service
      *
-     * @return The documentation that swagger shows on the browser.
+     * @return The documentation that swagger shows on the browser
      */
     @Bean
-    public Docket productApi() {
-        return new Docket(DocumentationType.SWAGGER_2)
-                .apiInfo(apiInfo())
+    public Docket apiDocket() {
+
+        Contact contact = new Contact(
+                "Jalasoft",
+                "http://www.jalasoft.com/",
+                "paolo.sandoval@jalasoft.com"
+        );
+
+        List<VendorExtension> vendorExtensions = new ArrayList<>();
+
+        ApiInfo apiInfo = new ApiInfo(
+                "Multilaguage Online Ide (MOI) app RESTful Web Service",
+                "This pages documents Multilaguage Online Ide app RESTful Web Service endpoints",
+                "1.0",
+                "http://www.jalasoft.com/terms", contact,
+                "Apache 2.0", "http://www.jalasoft.com/privacy",vendorExtensions);
```

Comment:
please, format your code.

Reply:
I think its fine because in our code conventions the first tab is four spaces an the second is eight spaces